t/venom/test_suites/pfcron/01_job_was_ran.yml

```diff
@@ -42,5 +56,14 @@ testcases:
   - type: exec
     script: 'cp "{{.pfserver_conf_dir}}/pfcron.conf.example" "{{.pfserver_conf_dir}}/pfcron.conf"'
 
+  - type: exec
+    script: 'cp "{{.fingerbank_conf_dir}}/fingerbank.conf.bak" "{{.fingerbank_conf_dir}}/fingerbank.conf"'
+
+  - type: exec
+    script: 'mv "{{.fingerbank_upstream_db}}.bak" "{{.fingerbank_upstream_db}}"'
+
   - type: exec
     script: '{{.pfserver_pfcmd}} configreload'
+
+  - type: exec
+    script: 'kill $(cat {{.pfserver_root_dir}}/var/run/fingerbank.pid)'
```

Comment:
Is it enough to have Fingerbank restarted with its configuration before test ?

Reply:
This is a fake service